frameworks/lightautoml/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+AMLB_DIR="$1"
+VERSION=${2:-"latest"}
```

Comment:
new requirements:
-  default is now 'stable' -> points to last release.
see other frameworks to show how it's handled, usually sth like:
```
if [[ "$VERSION" == "stable" ]]; then
    PIP install --no-cache-dir -U ${PKG}
elif [[ "$VERSION" =~ ^[0-9] ]]; then
    ...
```